addons/pause/userscript.js

```diff
@@ -15,6 +15,14 @@ export default async function ({ addon, console, msg }) {
   setSrc();
   onPauseChanged(setSrc);
 
+  if (addon.settings.get("auto-pause")) document.addEventListener("focusout", () => setPaused(true));
```

Comment:
I got this working by using `window` and `blur` instead of `document` and `focusout`.
```suggestion
  if (addon.settings.get("auto-pause")) window.addEventListener("blur", () => setPaused(true));
```